code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -416,6 +416,16 @@
 	desc = "Contains tramadol - a general use painkiller"
 	preloaded_reagents = list("tramadol" = 15)
 
+/obj/item/reagent_containers/syringe/paracetamol
+	name = "syringe (paracetamol)"
+	desc = "Contains paracetamol - a mild painkiller"
+	preloaded_reagents = list("paracetamol" = 15)
+
+/obj/item/reagent_containers/syringe/adrenaline
+	name = "syringe (adrenaline)"
+	desc = "Contains adrenaline"
+	preloaded_reagents = list("adrenaline" = 15)
```

Comment:
I don't think this is a good idea

Reply:
All contraband stims are humon's ideas
![image](https://user-images.githubusercontent.com/112724221/194382744-31792747-23be-461d-ab85-1c646f7fcf3c.png)